app/models/domain.rb

```diff
@@ -1,6 +1,14 @@
 require "resolv"
 # This models a DNS domain and so represents a site.
 class Domain < ApplicationRecord
+  extend ApipieDSL::Class
```

Comment:
This is already extended in `ApplicationRecord`, do we need to do this in every class?